Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue1758.cs

```diff
@@ -44,27 +56,55 @@ Layout Relative()
         
 			layout.Children.Add(_button, 
 				Forms.Constraint.Constant(0),
-				Forms.Constraint.Constant(300));
+				Forms.Constraint.Constant(0));
  
 			return layout;
 		}
 
 		Layout Absolute()
 		{
 			var layout = new AbsoluteLayout { Children = { _list, _button } };
- 
+
 			AbsoluteLayout.SetLayoutBounds(_list, new Rectangle(0, 0, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize));
 			AbsoluteLayout.SetLayoutBounds(_button, new Rectangle(0, 300, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize));
- 
+
 			return layout;
 		}
 
 		async void Animate()
 		{
-			// Comment this delay out to see the bug
-			// await Task.Delay(500);
- 
-			await _button.LayoutTo(new Rectangle(100, 100, 100, 100), 1000);
+			await Task.Delay(2000).ContinueWith(t => 
+			{
+				Device.BeginInvokeOnMainThread(async () =>
+				{
+					await _button.LayoutTo(new Rectangle(100, 100, 100, 100), 1000);
+				});
+			});
+		}
+
+#if UITEST
+		[Test]
+		public void CheckButtonPosition()
+		{
+			RunningApp.WaitForElement("Button");
+
+			var oldRect = RunningApp.Query("Button").Single().Rect;
+
+			try
+			{
+				RunningApp.WaitForNoElement("Button", timeout: TimeSpan.FromSeconds(5));
+			}
+			catch
+			{
+				// swallow exception
```

Comment:
Why is this needed?

Reply:
No, sure. Unfortunately, I don't think there is. But why is there an empty try/catch :)

Reply:
Sorry. Catching only `TimeoutException` now.